docs/source/app_developers_guide/about_events.rst

```diff
@@ -0,0 +1,123 @@
+*********************
+About Sawtooth Events
+*********************
+
+Sawtooth events occur "on block boundaries" - that is, when blocks are
+committed - and are not persisted in state. Each transaction family can define
+the events that are appropriate for its business logic.
+
+Events are represented with the following protobuf message:
+
+.. code-block:: python
```

Comment:
This isn't actually a python code block and so the comments look weird. It is probably fine to use python since the highlighting looks mostly accurate, but maybe try changing the comments to `#` so that the highlighting is more accurate.